frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -70,7 +70,8 @@ function CommonsForm({initialCommons, submitAction, buttonLabel = "Create"}) {
     
     const testid = "CommonsForm";
     const curr = new Date();
-    const today = curr.toISOString().split('T')[0];
+    const localTime = new Date(curr.getTime() - curr.getTimezoneOffset() * 60000);
+    const today = localTime.toISOString().split('T')[0];
     const currMonth = curr.getMonth() % 12;
     const nextMonth = new Date(curr.getFullYear(), currMonth + 1, curr.getDate()).toISOString().substr(0, 10);
     const DefaultVals = {
```

Comment:
I wonder if we could factor these lines of code into a method and put that method in a utility file?

It seems as if we have a lot of duplicate code here, and this is an opportunity to refactor it.